src/toolkits/object_detection/object_detector.cpp

```diff
@@ -579,6 +582,7 @@ void object_detector::perform_predict(gl_sframe data,
   augmenter_opts.batch_size = batch_size;
   augmenter_opts.output_height = grid_height * SPATIAL_REDUCTION;
   augmenter_opts.output_width = grid_width * SPATIAL_REDUCTION;
+  augmenter_opts.resize_only = true;
```

Comment:
The fact that this is resize-only is implied by the options being set. This flag is redundant